src/parser/demonhunter/havoc/modules/spells/BladeDance.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import SPECS from 'game/SPECS';
+import Events, { DamageEvent, FightEndEvent } from 'parser/core/Events';
+import { ThresholdStyle, When } from 'parser/core/ParseResults';
+import SpellLink from 'common/SpellLink';
+
+//For Blade dance and Death Sweep
+class BladeDance extends Analyzer {
+  get suggestionThresholds() {
+    return {
+      actual: this.badCast,
+      isGreaterThan: {
+        minor: 0,
```

Comment:
This probably shouldn't be 0 as you could get mechanics that require you to move, boss movement, or phasing that could make you miss a few hits. A little bit of lee way is a nice way to combat fight rng

Reply:
Typically if something is mess up once and get yelled at it has to be something that fight rng can't mess with

Reply:
I agree, the people who use this wrong will do it more than once, and if u have bad rng and it mess you up it should be minor. Updated the numbers.